.github/ISSUE_TEMPLATE/sprint_issue.md

```diff
@@ -27,6 +27,14 @@ Related spec: WIP
 - [ ] If prototype validated, merge changes into `main`
 - [ ] Update the spec
 
-## Impacted teams
```

Comment:
Can you let this section please? 😬

Reply:
yup sorry

Reply:
We could even pre-tick the documentation team as they will most certainly be concerned by a change in our API